Bluejay/Bluejay/Bluejay.swift

```diff
@@ -558,6 +563,9 @@ extension Bluejay: CBCentralManagerDelegate {
         UIApplication.shared.endBackgroundTask(backgroundTask)
     }
     
+    /**
+     Examine the listen cache in `UserDefaults` to determine whether there are any listens that might need restoration.
```

Comment:
@apike This is a private function intended for use by the framework only, and Jazzy doesn't render inline documentation for private members by default.

I think there are some benefits in documenting some implementation details for private functions, because it might come in handy when people want to fork or change internal behaviours. But for public API, we should definitely try to talk less about implementation details.

This particular function is also kind of generic and helper-like in nature, and therefore not a great candidate to apply any specific style of documentation – either way works. If you have something in mind, we can go with that, otherwise I don't have any immediate alternatives that are significantly better or worse than what's here.